csharp/BYOAuthFlow/Program.cs

```diff
@@ -0,0 +1,228 @@
+                "localhost",
+                SpatialdPort,
+                PlayerIdentityTokenServiceClient.Create(spatialdEndpoint),
+                LoginTokenServiceClient.Create(spatialdEndpoint),
+                DeploymentServiceClient.Create(spatialdEndpoint),
+                insecureConnection: true);
+        }
+
+        private static void RunScenario(string locatorHost, ushort locatorPort, PlayerIdentityTokenServiceClient pitClient, LoginTokenServiceClient ltClient,
+            DeploymentServiceClient dsClient, bool insecureConnection)
+        {
+            var deployment = Setup(dsClient);
+            try
+            {
+                var pit = CreatePIT(pitClient);
+                deployment = FindDeployment(dsClient, deployment.Id);
+                var loginToken = CreateLoginTokenForDeployment(ltClient, pit, deployment.Id);
+                ConnectToLocator(locatorHost, locatorPort, insecureConnection, pit, loginToken);
+            }
+            finally
+            {
+                Cleanup(dsClient, deployment);
+            }
+        }
+
+        private static string CreatePIT(PlayerIdentityTokenServiceClient pitClient)
+        {
+            Console.WriteLine("Generate a PIT");
+            var playerIdentityTokenResponse = pitClient.CreatePlayerIdentityToken(
+                new CreatePlayerIdentityTokenRequest
+                {
+                    Provider = "provider",
+                    PlayerIdentifier = "player_identifier",
+                    ProjectName = ProjectName,
+                });
+            return playerIdentityTokenResponse.PlayerIdentityToken;
+        }
+
+        private static Deployment FindDeployment(DeploymentServiceClient dsc, string deploymentId)
+        {
+            Console.WriteLine("Choosing deployment");
+            return dsc.ListDeployments(new ListDeploymentsRequest
+            {
+                ProjectName = ProjectName
+            }).First(d => d.Id == deploymentId);
+        }
+
+        private static string CreateLoginTokenForDeployment(LoginTokenServiceClient ltClient,
+            string pit, string deploymentId)
+        {
+            Console.WriteLine("Generate a Login Token for the selected deployment");
+            var createLoginTokenResponse = ltClient.CreateLoginToken(
+                new CreateLoginTokenRequest
+                {
+                    PlayerIdentityToken = pit,
+                    DeploymentId = deploymentId,
+                    LifetimeDuration = Duration.FromTimeSpan(new TimeSpan(0, 0, 30, 0)),
+                    WorkerType = "UnityClient",
+                });
+            var loginToken = createLoginTokenResponse.LoginToken;
+            return loginToken;
+        }
+
+        private static void ConnectToLocator(string locatorHost, ushort locatorPort, bool insecureConnection, string pit, string loginToken)
+        {
+            Console.WriteLine("Connect to the deployment using the Login Token and PIT");
```

Comment:
-> login token

Also, if you're going to use the abbreviation "PIT", you should probably introduce it the first time you mention the player identity token. E.g. "player identity token (PIT)"

Reply:
updated this to be PlayerIdentityToken and LoginToken though some variables may include 'pit' or 'lt'